### PR TITLE
move test bastion services to ignition and systemd and mirror docker images

### PR DESCRIPTION
**Pull request by enxebre**
cherry picks https://github.com/coreos/tectonic-installer/pull/2826

### CODE REVIEW

**Comment by cpanato** [comment]
retest this please

**Comment by cpanato** [comment]
error:
```
Error: Error applying plan:

3 error(s) occurred:

* module.dns.powerdns_record.api_internal: 1 error(s) occurred:

* powerdns_record.api_internal: Failed to create PowerDNS Record: Error creating record set: govvpcint-pr-2835-3bc0d5-api.tectonic-ci.de.:::ALIAS, reason: "Could not find domain 'tectonic-ci.de.'"
* module.dns.powerdns_record.etcd: 1 error(s) occurred:

* powerdns_record.etcd: Failed to create PowerDNS Record: Error creating record set: govvpcint-pr-2835-3bc0d5-etcd-0.tectonic-ci.de.:::A, reason: "Could not find domain 'tectonic-ci.de.'"
* module.dns.powerdns_record.ingress_private: 1 error(s) occurred:

* powerdns_record.ingress_private: Failed to create PowerDNS Record: Error creating record set: govvpcint-pr-2835-3bc0d5.tectonic-ci.de.:::ALIAS, reason: "Could not find domain 'tectonic-ci.de.'"
```

@enxebre

**Comment by cpanato** [comment]
All good now

File: contrib/govcloud/resources/gateway.service [hunk 1]
```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Nat Gateway routing
```
[the remark this answers is not in the document]
Comment (enxebre) [reply contrib/govcloud/resources/gateway.service]: Afaik there's no nat gateway service in govcloud https://docs.aws.amazon.com/govcloud-us/latest/UserGuide/using-services.html so some solution is needed for instances in a private subnet to connect to the internet

File: contrib/govcloud/README.md [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
+This directory contains Terraform configuration that provisions a VPC with a VPN connection and a PowerDNS server in AWS GovCloud.
+This is setup is not production ready and is designed to emulate a customer-like deployment in order to end-to-end test deploying Tectonic in AWS GovCloud using a required pre-created internal VPC.
 
 This Terraform configuration provisions the following AWS resources by default:
 * 1 VPC with name configured by `TF_VAR_vpc_name`
 * 4 subnets in the VPC with count configured by `TF_VAR_subnet_count`
 * 1 public subnet containing an internet gateway and NAT gateway
 * 1 private Route 53 zone for `tectonic.dev.coreos.systems` configured by `TF_VAR_base_domain`
-* 1 t2.micro EC2 Container Linux instance in the public subnet running docker containers for OpenVPN, PowerDNS and Nginx for serving OpenVPN client configuration.
+* 1 t2.micro EC2 Container Linux instance in the public subnet running docker containers for OpenVPN, PowerDNS (and mysql) and Nginx for serving OpenVPN client configuration.
```
[the remark this answers is not in the document]
Comment (enxebre) [reply contrib/govcloud/README.md]: injected as a variable now

**Pull request merged**